integration_tests/import_test.py

```diff
@@ -11,7 +11,9 @@
         "torch",
         "--extra-index-url https://download.pytorch.org/whl/cpu ",
     ),
-    "jax": ("jax[cpu]", ""),
+    # please update the jax version here if jax version is updated in
+    # requirements file
+    "jax": ("jax[cpu]==0.5.1 flax>=0.10.1", ""),
```

Comment:
I think we want two version here. We want to test that Keras can run on Jax without Flax installed at all (currently untested I think).

And we want to test that the basic flow can work with flax installed and nnx enabled (e.g. by setting the environment variable).

Reply:
cleaned this up. Kept the jax tests independent and separate